docs/arch/agent.md

```diff
@@ -0,0 +1,50 @@
+# Summary
+
+The agent is the powerhouse of SealCi. It receives actions and runs them in order to complete the operational part of the CI.
+
+# References
+
+## Actions
+
+A action is a defined as list of commands. actions are launched in a controlled environment, as defined by the agent configuration.
+
+## Artifacts
+
+Artifacts is the result of the execution of a action. We consider as artifacts:
+
+-  logs of execution of the pod
+-  any file that is created during the execution
```

Comment:
```suggestion
During their execution, actions generate artfacts, which include:

-  Execution logs
-  Files generated during the action execution
```

Reply:
You should not introduce the Kubernetes notion if pods here.

Reply:
Done